test/rules_test.js

```diff
@@ -32,6 +32,12 @@ describe('rules', () => {
         ['x^(--(x + 1))', 'x^(x + 1)']
     ])
 
+    suite('rearrange coefficient', rules.REARRANGE_COEFF, [
+        ['y^3 * 5', '5 y^3'],
+        ['yz * 3', '3 yz'],
+        ['3x^2 * 5', '5 (3 x^2)']
```

Comment:
I had a look at https://github.com/socraticorg/mathsteps/blob/1083124f163a196d0686677a63a264df41ea5b84/lib/checks/canRearrangeCoefficient.js.  It only supports rearrange products with two factors.  It's okay that we're a little more lenient but I think we should restrict things to having only a single constant factor so we don't have to deal with this case.